integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -60,8 +60,8 @@ spec:
       {{- end }}
       securityContext:
         fsGroup: {{ .Values.fsGroup }}
-      {{- if and $needJournalVolume (not $isRoot) }}
       initContainers:
+      {{- if and $needJournalVolume (not $isRoot) }}
```

Comment:
is this `chown` still needed now that we use local pv instead of hostPath? i would like to get rid of `runAsUser: 0` as that might be unavailable in security constrained environments

Reply:
Yea this should be removed.